files/usr/share/cinnamon/applets/menu@cinnamon.org/applet.js

```diff
@@ -2971,10 +2972,15 @@ class CinnamonMenuApplet extends Applet.TextIconApplet {
             }
         } else if (apps) {
             for (let i = 0; i < this._applicationsButtons.length; i++) {
-                if (apps.indexOf(this._applicationsButtons[i].app.get_id()) != -1) {
-                    this._applicationsButtons[i].actor.show();
+                let button = this._applicationsButtons[i];
+                let appId = button.app.get_id();
+                if (apps.indexOf(appId) != -1) {
+                    button.actor.show();
+                    if (appId == exactMatch) {
```

Comment:
Don't use `==`. Use `===` instead

Reply:
@clefebvre Can you make checks like these, e.g. not using `==`, a part of the CI/code style enforcement in javascript?

Reply:
Updating. FWIW, the existing code doesn't use ===/!==

Reply:
Yeah, I know. And those should definitively be fixed at some point (in a different PR). It is a buggy operator that should never be used, and is scary to change. Making it at least better in the area we touch goes a long way :-)